package.json

```diff
@@ -23,7 +25,8 @@
 		"test:plugins": "ts-mocha tests/plugins/**/*.ts",
 		"test:runner": "ts-mocha tests/testrunner-tests.ts",
 		"test": "npm-run-all test:*",
-		"tsc": "tsc && tsc -p tests/tsconfig.json"
+    "typecheck": "tsc --noEmit && tsc -p tests/tsconfig.json --noEmit",
+		"tsc": "tsc"
```

Comment:
We use a plain and simple `tsc` step to build dist/ output. The new `typecheck` script replaces the old `tsc` script for type checking without emitting output.

We should probably cover the other TS files as well, currently this covers only src/ and tests/.